<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDlg</class>
 <widget class="QWidget" name="FileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTitleWidget" name="ktitlewidget"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Required</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Filename:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="name">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" colspan="2">
        <layout class="QVBoxLayout" name="urlLayout"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Used Mirrors:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Recommended</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>File size (in bytes):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KRestrictedLine" name="size">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
           <property name="validChars">
            <string>0123456789</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Verification:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTreeView" name="used_hashes">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="KPushButton" name="add_hash"/>
         </item>
         <item>
          <widget class="KPushButton" name="remove_hash"/>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QFormLayout" name="formLayout_4">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="toolTip">
            <string extracomment="the basic identity of the metalink-file, e.g. KDE">The identity of the file</string>
           </property>
           <property name="text">
            <string>Identity:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="identity">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="toolTip">
            <string extracomment="the version of the file e.g. 4.3 for KDE">The version of the file</string>
           </property>
           <property name="text">
            <string>Version:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="version">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Seite">
      <attribute name="title">
       <string>Optional</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout_7">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="description">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Logo:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="logo">
           <property name="inputMask">
            <string/>
           </property>
           <property name="clickMessage">
            <string>Url to the logo</string>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="toolTip">
            <string>The language of the file</string>
           </property>
           <property name="text">
            <string>Language:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KComboBox" name="language"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Operating system:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KLineEdit" name="os">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Copyright:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="KLineEdit" name="copyright">
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_22">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>License:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="lic_name">
           <property name="clickMessage">
            <string>Name of the license</string>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Url:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="lic_url">
           <property name="clickMessage">
            <string>Url to the license</string>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_20">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Publisher:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KLineEdit" name="pub_name">
           <property name="toolTip">
            <string>publisher</string>
           </property>
           <property name="clickMessage">
            <string>Name of the publisher</string>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Url:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="KLineEdit" name="pub_url">
           <property name="toolTip">
            <string>publisher</string>
           </property>
           <property name="clickMessage">
            <string>Url to the publisher</string>
           </property>
           <property name="showClearButton" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KRestrictedLine</class>
   <extends>KLineEdit</extends>
   <header>krestrictedline.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>size</tabstop>
  <tabstop>add_hash</tabstop>
  <tabstop>remove_hash</tabstop>
  <tabstop>used_hashes</tabstop>
  <tabstop>identity</tabstop>
  <tabstop>version</tabstop>
  <tabstop>description</tabstop>
  <tabstop>logo</tabstop>
  <tabstop>language</tabstop>
  <tabstop>os</tabstop>
  <tabstop>lic_name</tabstop>
  <tabstop>lic_url</tabstop>
  <tabstop>pub_name</tabstop>
  <tabstop>pub_url</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
